### PR TITLE
fix set_current_tab: Indexp_current=-1 out of size

### LINKED ISSUE #12997: ERROR: set_current_tab: Indexp_current=-1 out of size
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Any, current master branch (ebbe2bd)

**Issue description:**
When opening a project on the current master, this error appears:
```
ERROR: set_current_tab: Indexp_current=-1 out of size (get_tab_count()=0)
   At: scene/gui/tab_container.cpp:403.
ERROR: set_current_tab: Indexp_current=-1 out of size (get_tab_count()=0)
   At: scene/gui/tab_container.cpp:403.
```

**Steps to reproduce:**
- Open any project, whether empty or with a simple main scene, the same error appears.

### PR DESCRIPTION
**Pull request by omar25h**
fixes #12997 
My first PR 🎉, hopefully I didn't mess things up 😅

### CODE REVIEW

File: editor/plugins/script_editor_plugin.cpp [hunk 1]
```diff
@@ -1584,25 +1584,27 @@ void ScriptEditor::_update_script_names() {
...
-		int new_cur_tab = -1;
-		int new_prev_tab = -1;
-		for (int i = 0; i < sedata.size(); i++) {
-			tab_container->move_child(sedata[i].ref, i);
-			if (new_prev_tab == -1 && sedata[i].index == prev_tab) {
-				new_prev_tab = i;
-			}
-			if (new_cur_tab == -1 && sedata[i].index == cur_tab) {
-				new_cur_tab = i;
+		if (!sedata.empty()) {
```
Comment (vnen) [inline editor/plugins/script_editor_plugin.cpp]: This could be an extra condition in the previous `if` instead of another nested `if` block.

Comment (omar25h) [reply editor/plugins/script_editor_plugin.cpp]: it's updated now

**Comment by akien-mga** [comment]
Congrats for your first PR :)

**Comment by omar25h** [comment]
Thanks :D

**Pull request merged**